examples/js/loaders/sea3d/SEA3D.js

```diff
@@ -4749,7 +4159,7 @@ SEA3D.EventDispatcher.prototype = {
 
 };
 
-SEA3D.EventDispatcher.apply = function ( object ) {
+SEA3D.EventDispatcher.apply = function( object ) {
 
 	object.addEventListener = SEA3D.EventDispatcher.prototype.addEvenListener;
```

Comment:
Thanks @tschw . I updated EventDispatcher at https://github.com/mrdoob/eventdispatcher.js/ and some other things with Object.assign. Nice!